Source/WebCore/Configurations/WebCore.xcconfig

```diff
@@ -23,6 +23,8 @@
 
 #include "FeatureDefines.xcconfig"
 #include "../../../Configurations/Version.xcconfig"
+#include? "<DEVELOPER_DIR>/AppleInternal/XcodeConfig/AppleFeatures.xcconfig"
```

Comment:
This is not the right place to discuss it in detail, but I very much don't understand how this file can work, given its global scope.